.evergreen/orchestration/drivers_orchestration.py

```diff
@@ -635,36 +635,83 @@ def start(opts):
     LOGGER.info("Starting mongo-orchestration... done.")
 
 
+import psutil
+
+
+def shutdown_proc(proc: psutil.Process) -> None:
+    proc.terminate()
+    if proc.wait(1) is None:
+        try:
+            proc.kill()
+        except psutil.NoSuchProcess:
+            pass
```

Comment:
```suggestion
	try:
		proc.terminate()
		try:
		    proc.wait(10) # Wait up to 10 seconds.
		except psutil.TimeoutExpired:
		    proc.kill()
	except psutil.NoSuchProcess:
		pass
```

A 1 second timeout seems really short. This may throw psutil.TimeoutExpired on failure to terminate the process within 1 second. Docs also state:

> On UNIX: If PID is not a child of os.getpid (current process) just wait until the process disappears and return None. If PID does not exist return None immediately.

which doesn't seem consistent with the current check.

Should this instead be implemented as suggested?